### ISSUE TITLE
Proto Polymorphism Support

### ISSUE THREAD

**Issue by Jay-Krish** [drslump/Protobuf-PHP#37]
I use extensions in proto like,
message Cat
{    extend Animal
    {
        required Cat animal = 100; // Unique Animal extension number
    }
    optional bool declawed = 1;
}
Potobuf-PHP generate successfully, and addExtension() also there
But it seems not working while codec, instead only gives me the rest of the message without extensions.

Just want to clarify, does Protobuf-PHP support codec of protos with extensions?

[ I have done with http://www.indelible.org/ink/protobuf-polymorphism/ and it worked well with Java, now using the same proto to setup  PHP project ]

**Comment by alexodev** [comment]
Yeah i need know too how can i set the extensions with PHP. i try using setExtension and AddExtension but when serialize nothing happens.
